lib/erb_lint/cli.rb

```diff
@@ -31,15 +31,14 @@ def run(args = ARGV)
       dupped_args = args.dup
       load_options(dupped_args)
 
-      if cache? && autocorrect?
-        failure!("cannot run autocorrect mode with cache")
-      end
-
       @files = @options[:stdin] || dupped_args
 
       load_config
 
-      cache_dir = @options[:cache_dir]
+      if cache? && autocorrect?
+        failure!("cannot run autocorrect mode with cache")
```

Comment:
Let's not fail here. It was a bit annoying when options were passed on the command line but with the options in the config file, it will be even less easy to get out of the problem.
The cache shouldn't be enabled, and we should also handle the clear cache option.